tests/components/whirlpool/__init__.py

```diff
@@ -1,19 +1,22 @@
 """Tests for the Whirlpool Sixth Sense integration."""
 from homeassistant.components.whirlpool.const import DOMAIN
-from homeassistant.const import CONF_PASSWORD, CONF_REGION, CONF_USERNAME
+from homeassistant.const import CONF_BRAND, CONF_PASSWORD, CONF_REGION, CONF_USERNAME
 from homeassistant.core import HomeAssistant
 
 from tests.common import MockConfigEntry
 
 
-async def init_integration(hass: HomeAssistant, region: str = "EU") -> MockConfigEntry:
+async def init_integration(
+    hass: HomeAssistant, region: str = "EU", brand: str = "Whirlpool"
+) -> MockConfigEntry:
     """Set up the Whirlpool integration in Home Assistant."""
     entry = MockConfigEntry(
         domain=DOMAIN,
         data={
             CONF_USERNAME: "nobody",
             CONF_PASSWORD: "qwerty",
             CONF_REGION: region,
+            CONF_BRAND: brand,
```

Comment:
We should also test `init_integration` without `CONF_BRAND:` verifying "update" works.

Reply:
Good call, I've added `test_setup_brand_fallback` in `test_init.py` to test that setup without a brand uses Whirlpool